### ISSUE TITLE
weird sorting

### ISSUE THREAD

**Issue by randomusername3** [AdrienPoupa/VinylMusicPlayer#185]
**Describe the bug**
I seem to have a few weird errors on Vinyl's artist page primarily.
I'm using a google Pixel XL with LineageOS 16 with MicroG, version 0.22.1 from F-Droid.

Randomly (it appears to me), some artists are in the wrong place.  for example, Gorillaz is located in the C's in between the bands The Black Angels and Boys Noize.  When I scroll using the bar, the bubble even reads out 'G' briefly.  This isn't the only one either, Marilyn Manson is located between The Crystal Method and Danger Doom and The Prodigy is in between Kyoty and Knives Out.  I'm sure there are others too. 

Is this mostly a case of some errant tag problems? or is is a sorting problem?

Screenshot of what I mean:
![Screenshot_Vinyl_Music_Player](https://user-images.githubusercontent.com/52719452/60921307-2d311280-a268-11e9-8c85-7e2096b903da.png)

thank you for your time working on this project.

**Comment by gkeegan** [comment]
I couldn't replicate any of what you mentioned, so I assume it is a tagging problem.

**Comment by randomusername3** [comment]
I think I figured it out:
If an artist beginning with M has one song on an album with an additional artist whos begins with J, for whatever reason it will place the album in J, rather than M.

For example.  I have one album by the Prodigy: The Fat of the Land.  On this one album, the 3rd song: Diesel Power was cowritten by Kool Keith and the tag showed under artist: "Kool Keith; The Prodigy".  This put the entire artist nestled under K rather than P.  I changed the order around and the artist jumped where the should back in P.  This is despite that every other song on the album is credited to "The Prodigy" under the artist tag.

This means for Gorillaz, for whom I have 6 albums, I'll have to dig through each song's tags to find out which one's doing it.  It's not the worst thing in the world, but do with this bit of information what you will.
